circle.yml

```diff
@@ -0,0 +1,27 @@
+machine:
+  node:
+    version: 7
+  pre:
+    - mkdir ~/.yarn-cache
+deployment:
+  staging:
+    branch: master
+    owner: artsy
+    commands:
+      - NODE_ENV=production make deploy env=staging
+  production:
+    branch: release
+    owner: artsy
+    commands:
+      - NODE_ENV=production make deploy env=production
```

Comment:
Right. Wonder which is better -- to use the CIRCLE_SHA or pull a sha with git? https://github.com/artsy/positron/blob/master/Makefile#L52